.github/workflows/lint.yml

```
@@ -0,0 +1,91 @@
+---
+name: VM lint workflow
+on:
+  pull_request:
```

I always forget that YAML allows this syntax. Is this typical of GitHub Actions definitions? 

```suggestion
  pull_request: null
```

is a little more explicit. They should parse to the same thing.

---

Yeah, I did it as they demonstrate it: https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#on and haven't seen yamllint complain